### ISSUE TITLE
FR: Breakdown reading statistics in sessions

### ISSUE THREAD

**Issue by jospalau** [koreader/koreader#10052]
You can see the total time devoted and pages read for every book in each day and you can see how it is distributed during that day if you do several reading sessions. 

It would be nice to have the statistics broken down to see time and pages per session. For instance tapping in the total time and listing the sessions or tapping in the individual bars of each session.

Thanks.

**Comment by NiLuJe** [comment]
That's sorta what the new vertical day view does, isn't it?

**Comment by jospalau** [comment]
Yes, but as far I see, no statistics for the sessions? It would be nice having them. If I read a few sessions a day I could see how long each. Not something important though.

**Comment by poire-z** [comment]
> If I read a few sessions a day I could see how long each

We don't remember statistics for "sessions", because "sessions" are not a thing :) There is just a "current session", which is everything that happens since some `self.start_current_period`.
Storing everything that happened in "past current session" feels complicated and quite useless practically.
@NiLuJe was mentionning the recently added Today's timeline (that you can get for past days from the calendar view), which shows you when and for how long you read (but ok, not how many pages you read in each of these durations, but does that really matter?):
ie. in the first screenshot at https://github.com/koreader/koreader/pull/9813#issuecomment-1320963008, I can see I had read for somehow 6 _nearly_ continuous periods:
![image](https://user-images.githubusercontent.com/24273478/202869378-d1209a06-050e-42e5-9b99-8f2a14b7f7ae.png)

**Comment by jospalau** [comment]
Good to know. I didn't know this functionality was done so recently. I have been using KOReader for 2 years and I just started using the statistics 2 weeks ago. It is really cool.

And you are right. As far as I can track time and total pages a day it is good and I can see the "sessions" visually. It still would be nice to tap in the periods and get the time but you can eyeball it.

Cheers!

**Comment by jospalau** [comment]
After having been using the statistics for a few days it does not make sense to track sessions. I  would find very useful to have the current session time in the Status Bar. That way I see when I started reading.

**Comment by NiLuJe** [comment]
> I would find very useful to have the current session time in the Status Bar.

This is somewhat similar to what early FW 3.x did on Kindle, IIRC.

(Well, not exactly, it just added a notch to the progress bar at the starting position).

It *may* have been discussed in the past (emphasis on *past* ;p).

**Comment by jospalau** [comment]
> > I would find very useful to have the current session time in the Status Bar.
> 
> This is somewhat similar to what early FW 3.x did on Kindle, IIRC.
> 
> (Well, not exactly, it just added a notch to the progress bar at the starting position).
> 
> It _may_ have been discussed in the past (emphasis on _past_ ;p).

I saw you made a marker to the progress bar in last night build. It is very useful. What I mean is to have an optional Time spent reading this session in the progress bar. I have a gesture to see the statistics but it would be nice to have the time handy
